rts/Lua/LuaSyncedRead.cpp

```diff
@@ -8295,6 +8281,34 @@ int LuaSyncedRead::IsUnitInJammer(lua_State* L)
 	return 1;
 }
 
+/***
+ *
+ * @function Spring.IsFeatureInLos
+ * @param featureID integer
+ * @param allyTeamID integer
```

Comment:
Reading [GetEffectiveLosAllyTeam](https://github.com/beyond-all-reason/RecoilEngine/blob/3d9b90e589e1dc0f342c1027a6f9d820e07b3753/rts/Lua/LuaSyncedRead.cpp#L8060-L8077).

It seems to me:

```suggestion
 * @param allyTeamID integer? Defaults to the current `allyTeamID`.
```

I wonder what happens when this argument is omitted in synced, `CLuaHandle::GetHandleReadAllyTeam` is a little inscrutable for me to understand (uses a macro and checks some context in the lua environment).

Many other such callouts that require `allyTeamID` should have their signatures adjusted as well.